docs/config/inventory.lst

```diff
@@ -3,12 +3,18 @@
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/custom_config.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/dns.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/external.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/fq_driver.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/ganglia.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/graphite_plain.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/graphite_tls.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/handoff_ingestor.xml"/>
```

Comment:
This is a stratcond module and is already included in that section, so it is not needed here.

Reply:
done